liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/PackageServerMojo.java

```diff
@@ -77,12 +89,13 @@ protected void doExecute() throws Exception {
         ServerTask serverTask = initializeJava();
         copyConfigFiles();
         serverTask.setOperation("package");
+        String fileType = minificationType(include);
         if (packageFile != null) {
             if (packageFile.isDirectory()) {
-                packageFile = new File(packageFile, serverName + ".zip");
+                packageFile = new File(packageFile, serverName + fileType);
```

Comment:
shouldn't this use projectBuildName too?  I.e., the only difference is the directory it goes in?

packageFile = new File(packageFile, projectBuildName + fileType);